README.md

```diff
@@ -1,344 +1,57 @@
-	- [track](#track)
-	- [identify](#identify)
-	- [screen](#screen)
-	- [group](#group)
-	- [add](#add)
-	- [find](#find)
-	- [remove](#remove)
-	- [flush](#flush)
-- [Plugin Architecture](#plugin-architecture)
-	- [Fundamentals](#fundamentals)
-	- [Advanced Concepts](#advanced-concepts)
-- [Contributing](#contributing)
-- [Code of Conduct](#code-of-conduct)
-- [License](#license)
+## How to get started
+1. **Collect analytics data** from your app(s).
+  - The top 200 Segment companies collect data from 5+ source types (web, mobile, server, CRM, etc.).
+2. **Send the data to analytics tools** (for example, Google Analytics, Amplitude, Mixpanel).
+  - Over 250+ Segment companies send data to eight categories of destinations such as analytics tools, warehouses, email marketing and remarketing systems, session recording, and more.
+3. **Explore your data** by creating metrics (for example, new signups, retention cohorts, and revenue generation).
+  - The best Segment companies use retention cohorts to measure product market fit. Netflix has 70% paid retention after 12 months, 30% after 7 years.
 
-## Installation
-Add the Swift package as a dependency either via your package.swift, or via Xcode's File->Swift Packages->Add Package Dependency menu item.
+[Segment](https://segment.com) collects analytics data and allows you to send it to more than 250 apps (such as Google Analytics, Mixpanel, Optimizely, Facebook Ads, Slack, Sentry) just by flipping a switch. You only need one Segment code snippet, and you can turn integrations on and off at will, with no additional code. [Sign up with Segment today](https://app.segment.com/signup).
 
-`git@github.com:segmentio/analytics-swift.git`
+### Why?
+1. **Power all your analytics apps with the same data**. Instead of writing code to integrate all of your tools individually, send data to Segment, once.
 
-Once completed, Analytics can be referenced by importing Segment's Analytics package
+2. **Install tracking for the last time**. We're the last integration you'll ever need to write. You only need to instrument Segment once. Reduce all of your tracking code and advertising tags into a single set of API calls.
 
-`import Segment`
+3. **Send data from anywhere**. Send Segment data from any device, and we'll transform and send it on to any tool.
 
-## Usage
-### Setting up the client
-The Analytics client will typically be set up at application launch time, such as `applicationDidFinishLaunching`.
+4. **Query your data in SQL**. Slice, dice, and analyze your data in detail with Segment SQL. We'll transform and load your customer behavioral data directly from your apps into Amazon Redshift, Google BigQuery, or Postgres. Save weeks of engineering time by not having to invent your own data warehouse and ETL pipeline.
 
-Typically the following call may be all that's required.
+   For example, you can capture data on any app:
+    ```kotlin
+    analytics.track('Order Completed', Properties(price = 99.84))
```

Comment:
This line is incorrect. Not sure where `Properties` is coming from so hard to discuss how to fix.

Reply:
It's also mentioning Kotlin in the Swift repo.  Examples of this can be found in Stacy's docs.